app/javascript/components/explore/ExploreDisplayTabs.jsx

```diff
@@ -198,6 +198,23 @@ function getCellFacetingData(cluster, annotation, setterFunctions, context, prev
   }
 }
 
+/** Determine whether to enable cell filtering tab for ATAC data */
+function shouldAllowIgvFiltering(shownTab, exploreInfo) {
+  const flags = getFeatureFlagsWithDefaults()
+
+  // Consider refining upload UI or processing to more specifically
+  // determine if BED file is for _multiome_.  Users could theoretically
+  // upload non-multiome BED, though that's unlikely and showing this
+  // functionality in that case is low impact.
+  const hasBedGenomeFiles = exploreInfo && exploreInfo?.bedBundleList?.length > 0
+
+  const allowIgvFiltering =
```

Comment:
Tiny nit - you could just return this w/o declaring a const, correct?

Reply:
Yup!  I generally like to assign a variable for a function's return value, because I find it easier to read and debug.